chart/jenkins-operator/templates/jenkins.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
   {{- with .Values.jenkins.labels }}
   labels: {{ toYaml . | nindent 4 }}
   {{- end }}
+  {{- with .Values.jenkins.annotations }}
+  annotations: {{ toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
The annotations should be configurable only for Jenkins pod, not Jenkins instance. Please revert this change.